bin/wabur

```diff
@@ -129,15 +129,24 @@ options = {
 
 config = WAB::Impl::Configuration.new(usage, options)
 mode = config[:mode]
+# TBD: sanitize path to ensure ancestor-level directories cannot be written to.
+dir = File.expand_path(config[:base] || '.')
 case mode
 when 'new'
-  dir = File.expand_path(config[:base] || '.')
-  abort(WAB::Impl.format_error("#{dir} already exists. Refusing to over-write. Use init instead.")) if Dir.exist?(dir)
+  if Dir.exist?(dir)
```

Comment:
The `dir = File.expand_path(config[:base] || '.')` line is still needed to handle the case where the user does not specify a base.

Reply:
yes.. It's being assigned on a line above..